linkerd/app/outbound/src/tcp/tests.rs

```diff
@@ -296,7 +296,7 @@ async fn load_balances() {
         .map(|i| {
             tokio::spawn(
                 hello_world_client(svc_addr, &mut server)
-                    .instrument(tracing::info_span!("conn", i)),
+                    .instrument(tracing::debug_span!("conn", i)),
```

Comment:
IMO it's nice to have the spans in these tests on by default, was the change to the tests intended, or was it just an accident of find-and-replace?

Reply:
yeah, global replace. restored test spans.